src/common/cluster-store.ts

```diff
@@ -17,9 +17,18 @@ export interface ClusterIconUpload {
   path: string;
 }
 
+interface ClusterIconOrdering {
+  [workspaceId: string]: ClusterId[]
+}
+
+interface ClusterIconOrderingMap {
+  [id: string]: number
+}
```

Comment:
Why need this? `ClusterIconOrdering` only should be enough to keep clusters order?